### PR TITLE
Fixed a problem where if slur and tie stops are in separate notations…

### PR DESCRIPTION
**Pull request by johanvelthuis**
… tags, the slur end would be missed, resulting in endless slurs on notes with both a slur end and a tie stop. According to the musicxml specs multiple notations tags are allowed, so they should be dealt with.
[Say Something SSAB - Full score - 01 0 tutti Pentatonix - Say Something SSAB.musicxml.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/10928500/Say.Something.SSAB.-.Full.score.-.01.0.tutti.Pentatonix.-.Say.Something.SSAB.musicxml.zip)

In this example file, look at the bottom staff. Every 4 measures there should be a slur. In version 1.7.4, when a note has a tie end and a slur end, the slur end is ignored, which the
Schermafbeelding 2023-03-09 om 07 40 12
 slur longer until another note has a slur end that is recognized.

Below is an image of the new situation.
Schermafbeelding 2023-03-09 om 07 41 03

### CODE REVIEW

File: src/Common/FileIO/Xml.ts
Comment (sschmidTU) [inline src/Common/FileIO/Xml.ts:117]: please add a space after the if.

File: src/Common/FileIO/Xml.ts
Comment (sschmidTU) [inline src/Common/FileIO/Xml.ts:122]: please use our code style: space after the if and before the curly bracket

File: src/Common/FileIO/Xml.ts
Comment (sschmidTU) [inline src/Common/FileIO/Xml.ts:128]: code style: space after ending curly brace and before opening curly bracket

File: src/Common/FileIO/Xml.ts
Comment (sschmidTU) [inline src/Common/FileIO/Xml.ts:133]: spaces

**Review by sschmidTU — CHANGES_REQUESTED** [review]
Looks good to me, except for the few code style change requests.
Once these are adapted, we can merge.

No visual regressions (see [Wiki | Testing](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/wiki/Testing)).

I'll add a test sample with the first 5 measures of the sample in your PR comment. I'm guessing the full sample might be copyrighted, but in any case, the first 5 measures should be fine.

(you can merge the code style changes into your new PR #1342 so they are also reflected there, since you didn't start the PR from develop)

**Comment by sschmidTU** [comment]
You can then merge the code style changes into your new PR #1342 so they are also reflected there (since you didn't start that PR from develop).
Either merge it yourself from your own branch, or wait until this PR is merged into develop and then merge develop into your other branch.

**Comment by sschmidTU** [comment]
@johanvelthuis Please do the code style changes in this PR/branch first, so we can merge it and you can merge it into your other PR #1342 :)
I can also do this after merging, but then you won't get credit for the changed lines.

File: src/Common/FileIO/Xml.ts
Comment (johanvelthuis) [reply src/Common/FileIO/Xml.ts]: done

File: src/Common/FileIO/Xml.ts
Comment (johanvelthuis) [reply src/Common/FileIO/Xml.ts:127]: done

File: src/Common/FileIO/Xml.ts
Comment (johanvelthuis) [reply src/Common/FileIO/Xml.ts]: done

**Comment by johanvelthuis** [comment]
@sschmidTU I tried to conform this PR to the correct coding style.

File: src/Common/FileIO/Xml.ts
Comment (johanvelthuis) [reply src/Common/FileIO/Xml.ts]: done

File: src/Common/FileIO/Xml.ts
Comment (sschmidTU) [inline src/Common/FileIO/Xml.ts:127]: one more spacing difference here, but I'll fix that after the merge.

**Comment by sschmidTU** [comment]
Merged, thank you very much for the fix and contribution, and the patience with change requests @johanvelthuis !
This is a very nice fix for Dorico files.
You are right, we don't have a lot of Dorico samples, as we mostly use MuseScore for export, though we do also have some Sibelius-exported samples added.
We could definitely add more samples like this that test unique syntax cases.

**Pull request merged**
